public/index.html

```diff
@@ -62,6 +80,28 @@ <h2>Part II</h2>
   "description": "Thanks for the laughs!",
   "status": "initiated",
   "initiatedAt": "2019-05-30T12:53:38-04:00" // For an extra challenge, set this value to the current time
+}
+            </code>
+          </pre>
+        </div>
+        <div class="row">
+          <p>
+            This method should output the response from the Transfers API. For example:
+          </p>
+        </div>
+        <div class="row">
+          <pre style="width:100%">
+            <code class="json">
+{
+  "id": 1,
+  "status": "complete",
+  "originAccount": 111,
+  "targetAccount": 212,
+  "amount": 10000,
+  "description": "An example of a completed transfer",
+  "initiatedAt": "2019-05-20T12:53:38-04:00",
+  "completedAt": "2019-05-22T12:00:00-04:00",
+  "failedAt": null
```

Comment:
Can we have this example output aligned with the example input? And even with an `id` higher than what's included in the base data?